src/org/zaproxy/zap/control/AddOn.java

```diff
@@ -54,6 +57,14 @@
 import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
 
 public class AddOn  {
+
+	/**
+	 * The name of the manifest file, contained in the add-ons.
+	 * 
+	 * @since TODO add version
+	 */
+	public static final String MANIFEST_FILE_NAME = "ZapAddOn.xml";
+
 	public enum Status {unknown, example, alpha, beta, weekly, release}
 	
 	private static ZapRelease v2_4 = new ZapRelease("2.4.0");

```

Comment:
Is this supposed to be updated per release? Aren't version numbers also maintained in a constant class somewhere? (I thought I remembered seeing them elsewhere)